Assets/MixedRealityToolkit/Boundary/Scripts/BoundaryManager.cs

```diff
@@ -3,12 +3,10 @@
 
 using MixedRealityToolkit.Common;
 using UnityEngine;
-
-#if UNITY_WSA && UNITY_2017_2_OR_NEWER
```

Comment:
Why remove these?

Reply:
This all works in the editor so I didn't see any reason to restrict it in UNITY_WSA.

However, I did just find out we aren't abandoning older versions of Unity yet.  So I will just leave this PR blocked until we can just move to >= 2017.2 variants.

Reply:
It's no matter, vNEXT will most likely pick up from there and you've adding it into the dev branch, so I don't see any issues there.

Reply:
The `UNITY_WSA` define does run in the editor, if UWP is selected as the build type.

Reply:
Simply put @KevinKennedy , builds have to work for BOTH UWP and Standalone.  So that consumers can utilise other frameworks for VR builds.
If it doesn't build standalone, it can't be merged.

Reply:
Ha, it was news to me that the MRTK supports non-MR targets.  I'll add that back in.

Reply:
Not that we support other non-MR targets (yet) but we cannot prevent users from using others with our asset in their project.